src/S3/Makefile

```diff
@@ -0,0 +1,10 @@
+.EXPORT_ALL_VARIABLES:
+
+# Build the PHP runtimes
+initialize:
+	docker pull lphoward/fake-s3
+	docker start async_aws_s3 || docker run -d -p 4569:4569 --name async_aws_s3 lphoward/fake-s3
```

Comment:
`docker rm -fv` to clear previous test ?

Reply:
I should probably add a "stop" method. But I want to "start all" so I can run my test one by one..